src/reducers/commentsReducer.js

```diff
@@ -0,0 +1,24 @@
+import { CREATE_COMMENTS, CREATE_COMMENTS_ERRORS } from "../actions/types";
+
+const initialState = {
+  success: {},
+  errors: {}
+};
+
+export default (state = initialState, action) => {
+  console.log("reducer");
```

Comment:
Please remove these console logs

Reply:
@anyatibrian , please remove the `console.log` statement.

Reply:
done